app/src/main/java/org/mozilla/fenix/session/PrivateNotificationService.kt

```diff
@@ -33,9 +37,48 @@ class PrivateNotificationService : AbstractPrivateNotificationService() {
 
     override fun NotificationCompat.Builder.buildNotification() {
         setSmallIcon(R.drawable.ic_private_browsing)
-        setContentTitle(applicationContext.getString(R.string.app_name_private_4, getString(R.string.app_name)))
-        setContentText(applicationContext.getString(R.string.notification_pbm_delete_text_2))
-        color = ContextCompat.getColor(this@PrivateNotificationService, R.color.pbm_notification_color)
+        setContentTitle(
+            applicationContext.getString(
+                R.string.app_name_private_4,
+                getString(R.string.app_name)
+            )
+        )
+        setContentText(
+            applicationContext.getString(
+                R.string.notification_pbm_delete_text_2
+            )
+        )
+        color = ContextCompat.getColor(
+            this@PrivateNotificationService,
+            R.color.pbm_notification_color
+        )
+    }
+
+    override fun notifyLocaleChanged(notificationTag: String, channelId: String) {
+        // how do we get the notification itself?
+
+        val notification = super.createNotification(channelId)
+
+        NotificationManagerCompat.from(applicationContext)
+            .notify(applicationContext, notificationTag, notification)
+    }
+
+    companion object {
+        private const val NOTIFICATION_TAG =
+            "mozilla.components.feature.privatemode.notification.AbstractPrivateNotificationService"
+        const val ACTION_ERASE = "mozilla.components.feature.privatemode.action.ERASE"
+
+        val NOTIFICATION_CHANNEL = ChannelData(
+            id = "browsing-session",
+            name = R.string.mozac_feature_privatemode_notification_channel_name,
+            importance = NotificationManagerCompat.IMPORTANCE_LOW
+        )
+
+        // List of Intent actions that will get ignored when they are in the root intent that gets
+        // passed to onTaskRemoved().
+        private val ignoreTaskActions = listOf(
+            "mozilla.components.feature.pwa.VIEW_PWA"
+        )
```

Comment:
I don't see where we are using `ignoreTaskActions` ? :)

Reply:
THis was a copy/paste from the abstract notification service. It's been removed